.github/workflows/R-CMD-check.yml

```diff
@@ -75,17 +65,10 @@ jobs:
           done < <(Rscript -e 'writeLines(remotes::system_requirements("ubuntu", "20.04"))')
 
       - name: Install dependencies
-        if: matrix.config.r == '3.6'
```

Comment:
I can see why 3.5 is removed, but is 3.6 supposed to be removed from this as well?  I've never done this so all new to me